pkg/scheduler/algorithm/priorities/requested_to_capacity_ratio.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package priorities
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
+	"k8s.io/kubernetes/pkg/scheduler/schedulercache"
+)
+
+type functionShape struct {
+	x []float64
+	y []float64
+	n int
+}
+
+var (
+	// give priority to mostly utilized nodes by default
+	defaultFunctionShape = newFunctionShape([]float64{0.0, 1.0}, []float64{0.0, 1.0})
+)
+
+func newFunctionShape(x []float64, y []float64) functionShape {
```

Comment:
Please merge this function with `newFunctionShapeErr` and call the result `newFunctionShape`. It is odd to have a function called `newFunctionShapeErr`